### PR TITLE
Fixes wrong strings interpolation

### PR DESCRIPTION
**Pull request by artem-zakharchenko**
Minor changes focused on cleaning up test suites and fixing CoffeeScript migration bugs.

### CODE REVIEW

File: test/unit/mixins/validatable-http-message-test.js [hunk 1]
```diff
@@ -1193,12 +1192,12 @@ Unexpected token '\\'' at 1:22
...
-                  results.forEach((result) => {
-                    results.push(result.severity);
-                  });
-
-                  assert.include(results, 'error');
+                  const severities = instance.validation.body.results.map(
+                    (result) => {
+                      return result.severity;
+                    }
+                  );
```
Comment (honzajavorek) [inline test/unit/mixins/validatable-http-message-test.js]: Might be nicer:

```js
const severities = instance.validation.body.results.map(result => result.severity);
```

File: test/unit/mixins/validatable-http-message-test.js [hunk 2]
```diff
@@ -1299,7 +1298,8 @@ Unexpected token '\\'' at 1:22
 
             describe('when a validator throws an error', () => {
               before(() => {
-                invalidSchema = require('../../fixtures/invalid-schema-v4');
+                // eslint-disable-next-line global-require
+                const invalidSchema = require('../../fixtures/invalid-schema-v4');
```
Comment (honzajavorek) [inline test/unit/mixins/validatable-http-message-test.js]: Why not to require this at the top of the file, like

```js
const INVALID_SCHEMA = require('../../fixtures/invalid-schema-v4');
```

Comment (artem-zakharchenko) [reply test/unit/mixins/validatable-http-message-test.js]: To be honest, I have no idea. I think there was some intent to do it this way in the past, but I can't verify it.

File: test/unit/mixins/validatable-http-message-test.js (hunk 1 above)
Comment (artem-zakharchenko) [reply test/unit/mixins/validatable-http-message-test.js]: Good, will update it.

**Pull request merged**